TweetBook/Contracts/V1/ApiRoutes.cs

```diff
@@ -9,12 +9,15 @@ public static class ApiRoutes
         public const string Version = "v1";
 
         public const string Base = $"{Root}/{Version}";
+        // public const string Base = Root + "/" + Version;

```

Comment:
These lines are equivalent, you can remove the comment

Reply:
Because is a course of learning, please let them there, are for me to see the equivalence.